catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/UdpStreamMonitor.java

```diff
@@ -170,7 +171,7 @@ public UdpStreamMonitor() {
     UdpStreamMonitor(UdpStreamProcessor udpStreamProcessor) {
         this.udpStreamProcessor = udpStreamProcessor;
     }
-
+    
```

Comment:
I found that some of the visitors were no longer being used, so I was able to remove them.